src/components/Integrations/Table.tsx

```diff
@@ -202,19 +203,22 @@ const columns: Array<ICell> = [
     transforms: [],
   },
   {
-    title: (
-      <>
-        <span>
-          {Messages.components.integrations.table.columns.lastConnectionAttempt}
-        </span>
-        <LastConnectionHelpPopover>
-          <Button variant={ButtonVariant.plain}>
-            <HelpIcon />
-          </Button>
-        </LastConnectionHelpPopover>
-      </>
-    ),
-    transforms: [],
+    title: Messages.components.integrations.table.columns.lastConnectionAttempt,
+    transforms: [
+      info({
+        popover: <LastConnectionHelpTable />,
+        popoverProps: {
+          hasAutoWidth: true,
+          headerContent: (
+            <TextContent>
+              <Text component="h6">
+                {'Last connection attempt status meanings'}
```

Comment:
No need to wrap in string expression
```suggestion
                Last connection attempt status meanings
```